CHANGELOG-3.1.md

```diff
@@ -0,0 +1,25 @@
+# CHANGELOG for 3.1.x
+
+## 3.1.0 (2022-06-19)
+
+### PHP Versions
+
+  * 🗑️ Drop PHP 7.2 support
+  * ✔️ PHP 7.3
+  * ✔️ PHP 7.4
+  * 🚀 Add PHP 8.0
+  * 🚀 Add PHP 8.1
+  * 🧪 Set PHP 8.2 for nightly
+
+### Symfony Versions
+
+  * ✔️ Symfony 4.4
+  * ✔️ Symfony 5.4
+  * 🧪 Set Symfony 6.0 for nightly
```

Comment:
Codacy has a fix for the issue: [Expected: 0; Actual: 2](https://app.codacy.com/gh/yoanm/symfony-jsonrpc-http-server/pullRequest?prid=9640734)
```suggestion
* 🧪 Set Symfony 6.0 for nightly
```